gcp/main.tf

```diff
@@ -15,7 +15,25 @@ provider "kubernetes" {
   token            = data.google_client_config.provider.access_token
 }
 
-module "pd_backed_nfs_server" {
-  source     = "./pd-backed-nfs-server"
+provider "helm" {
+  kubernetes {
+    host = "https://${data.google_container_cluster.my_cluster.endpoint}"
+    cluster_ca_certificate = base64decode(
+    data.google_container_cluster.my_cluster.master_auth[0].cluster_ca_certificate,
+    )
+    token            = data.google_client_config.provider.access_token
+  }
+}
+
+resource "helm_release" "nfs_server_provisioner" {
```

Comment:
i think we should move it to other file, WDYT?

Reply:
IDK... It's just a single resource (and the only one on this entire tf stack), I think it'll be more straightforward considering this is a reference example